### PR TITLE
Weno fixes and updates

### PR DESCRIPTION
**Pull request by sjpadgett**
- Improve pharmacy import by using transactional queries. From 4 minutes to 40 seconds.
- improve validation and error reporting.
- some code cleanup for warnings and in general
- fix frame screen output to use the current theme so user always see content.

Fixes #

### CODE REVIEW

File: interface/modules/custom_modules/oe-module-weno/scripts/file_download.php [hunk 1]
```diff
@@ -105,87 +105,98 @@ function download_zipfile($fileUrl, $zipped_file)
...
-
-            ++$l;
         }
+    } catch (Exception $e) {
+        throw new RuntimeException($e->getMessage(), $e->getCode(), $e);
     }
     $wenolog->insertWenoLog("pharmacy", "Success");
     fclose($records);
+    sqlStatementNoLog('COMMIT');
+    sqlStatementNoLog('SET autocommit=1');
```
Comment (sjpadgett) [inline interface/modules/custom_modules/oe-module-weno/scripts/file_download.php:198]: I wonder if I should be concerned with default settings for PHP and engine resources with approx 100, 000 inserts?
Perhaps safer to batch maybe every 10,000 or so!

**Comment by sjpadgett** [comment]
That's it for this go around. Thanks SW for warning fix

**Comment by stephenwaite** [comment]
weno bueno :)
arlo-hello

File: interface/modules/custom_modules/oe-module-weno/src/Services/WenoPharmaciesImport.php [hunk 2]
```diff
@@ -27,94 +27,99 @@ public function importPharmacy($csvFile, $files)
...
-                    ++$l;
+                        ++$l;
+                    }
                 }
+            } catch (Exception $e) {
+                return $e->getMessage();
             }
             fclose($records);
+            sqlStatementNoLog('COMMIT');
+            sqlStatementNoLog('SET autocommit=1');
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-weno/src/Services/WenoPharmaciesImport.php:122]: Doesn't the following need to be placed above:
```
            sqlStatementNoLog('SET autocommit=0');
            sqlStatementNoLog('START TRANSACTION');
```

Comment (sjpadgett) [reply interface/modules/custom_modules/oe-module-weno/src/Services/WenoPharmaciesImport.php:122]: Nice catch @bradymiller and thank you!

**Pull request merged**